src/cli.rs

```diff
@@ -69,7 +69,7 @@ pub fn build_cli() -> App<'static, 'static> {
                         .short("o")
                         .long("output-dir")
                         .takes_value(true)
-                        .help("Outputs the generated site in the given path"),
+                        .help("Outputs the generated site in the given path (by default 'public' dir in project root)"),
```

Comment:
Thinking about that, `zola serve` doesn't actually outputs the generated site, only the assets. The HTML is kept in memory. I'll tweak the message later.

Reply:
Just leaving a note here: when you'll tweak the message for `--output-dir` in `zola serve`, also mention the same on https://www.getzola.org/documentation/getting-started/cli-usage/#serve

Currently it says nothing about output_dir only containing the assets and HTML being in memory.